src/fsharp/AccessibilityLogic.fs

```diff
@@ -323,8 +326,11 @@ let IsMethInfoAccessible amap m ad minfo = IsTypeAndMethInfoAccessible amap m ad
 
 let IsPropInfoAccessible g amap m ad = function 
     | ILProp ilpinfo -> IsILPropInfoAccessible g amap m ad ilpinfo
-    | FSProp (_, _, Some vref, _) 
-    | FSProp (_, _, _, Some vref) -> IsValAccessible ad vref
+    | FSProp (_, _, Some vref, None) 
+    | FSProp (_, _, None, Some vref) -> IsValAccessible ad vref
+    | FSProp (_, _, Some vrefGet, Some vrefSet) -> 
+        // pick most accessible
+        IsValAccessible ad vrefGet || IsValAccessible ad vrefSet
```

Comment:
So this makes a lot of sense. I feel like we need to do something similar on the `ILProp` side, maybe inside `IsILPropInfoAccessible`?

Reply:
@TIHan I think this is handled:  https://github.com/dotnet/fsharp/blob/1ccb4b1443445bb31f55e6426ea2c8c4b016fbae/src/fsharp/AccessibilityLogic.fs#L255-L265

it returns the max accessible in `GetILAccessOfILPropInfo` which is then checked against `IsILTypeAndMemberAccessible`.

Does that make sense?